### DISCUSSION TITLE
Allow creating items that is marked as non editable

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#3026] [Feature Requests]
Would be nice to also support an option to allow "editing" the field when creating an item so we can create items even if we are not able to edit them later.

**Comment by benhaynes** [comment]
Can't this be done with the CRUD permissions filter rules?

**Reply by rijkvanzanten** [reply]
Disabling the edit permissions for this particular field on the update action should do the trick

**Comment by connorwinston** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
